spec/features/users_can_edit_user_info_spec.rb

```diff
@@ -24,7 +24,7 @@
 
     allow_any_instance_of(ApplicationController)
       .to receive(:current_user)
-      .and_return(user)
+            .and_return(user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 8) spaces for indenting an expression spanning multiple lines.